database/migrations/2024_03_07_160308_remove_three_tables.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+return new class extends Migration
+{
+    /**
+     * Run the migrations.
+     */
+    public function up(): void
+    {
+        Schema::disableForeignKeyConstraints();
+
+        Schema::table('dimensions', function (Blueprint $table) {
+            $table->dropForeign('dimensions_topic_id_foreign');
+            $table->dropColumn('topic_id');
+        });
+
+        Schema::dropIfExists('topics');
+        Schema::dropIfExists('sub_dimensions');
+        Schema::dropIfExists('farming_systems');
```

Comment:
Could also remove the link tables (metric_topic, metric_sub_dimension, metric_farming_system)

Reply:
Good spot!